py/desisim/scripts/brightsims_aug2017.py

```diff
@@ -0,0 +1,137 @@
+"""
+desisim.scripts.brightsims
```

Comment:
OK to add this file for now, but we should be careful about the boundary between generally useful scripts (ok for this repo) vs. one-off scripts or functions that are tied to a particular analysis (better in a different repo custom for that analysis).  Proceed with this but we may refactor later when we know more about what is generally useful vs. bespoke.